### PR TITLE
Bump cosmos version from 0.45.4 to 0.45.9

### PR DESCRIPTION
**Pull request by Kbhat1**
## Describe your changes and provide context
- We want to bump our cosmos sdk fork version from 0.45.4 to 0.45.9. 
- This is the first PR in a set that incrementally adds the changes to 0.45.5
- Based on exactly the changes here: https://github.com/cosmos/cosmos-sdk/compare/v0.45.4...v0.45.5
## Testing performed to validate your change
- Ran updated tests and chain

### CODE REVIEW

**Review by codchen — APPROVED** [review]

File: baseapp/abci.go [hunk 1]
```diff
@@ -795,35 +819,6 @@ func handleQueryStore(app *BaseApp, path []string, req abci.RequestQuery) abci.R
 	return resp
 }
 
-func handleQueryP2P(app *BaseApp, path []string) abci.ResponseQuery {
```
Comment (codchen) [inline baseapp/abci.go:798]: we need to keep this since it's used by `Query`

File: baseapp/abci.go [hunk 2]
```diff
@@ -130,24 +131,6 @@ func (app *BaseApp) Info(ctx context.Context, req *abci.RequestInfo) (*abci.Resp
 	}, nil
 }
 
-// FilterPeerByAddrPort filters peers by address/port.
```
Comment (codchen) [inline baseapp/abci.go:133]: need to keep these two

File: baseapp/abci.go (hunk 1 above)
Comment (Kbhat1) [reply baseapp/abci.go:798]: This was moved to `baseapp/p2p.go`. Is that still an issue?

File: baseapp/abci.go (hunk 2 above)
Comment (Kbhat1) [reply baseapp/abci.go:133]: As above, this was moved to `baseapp.p2p.go`

**Review by philipsu522 — APPROVED** [review]

**Pull request merged**